xml/System.IO.Compression/BrotliEncoder.xml

```diff
@@ -60,9 +60,7 @@
         <remarks>To be added.</remarks>
         <exception cref="T:System.ArgumentOutOfRangeException">
           <paramref name="quality" /> is not between the minimum value of 0 and the maximum value of 11.
-
```

Comment:
Removing the endlines that surround an -or-` causes MS Docs to show everything into one line. The tool needs to preserve empty lines.

https://review.docs.microsoft.com/en-us/dotnet/api/system.io.compression.brotliencoder.-ctor?view=net-5.0&branch=pr-en-us-5213

![image](https://user-images.githubusercontent.com/1175054/104226817-3e2e8700-53fd-11eb-9713-757028c9936a.png)